spras/analysis/ml.py

```diff
@@ -55,7 +55,10 @@ def summarize_networks(file_paths: Iterable[Union[str, PathLike]]) -> pd.DataFra
                         # node order does matter for directed edges
                         edges.append(DIR_CONST.join([node1, node2]))
                     else:
-                        ValueError(f"direction is {direction}, rather than U or D")
+                        if direction == 'Direction': # if reading the header
```

Comment:
This looks like it can be combined into an `elif` so we don't have the additional level of nesting.